FIX make sure to clone remainder in TableVectorizer (#678)
skrub-data/skrub: CHANGES.rst

  compliance with the scikit-learn API.
  :pr:`647` by :user:`Guillaume Lemaitre <glemaitre>`

> * Fixes a bug in :class:`TableVectorizer` where `remainder` is now cloned if this is
>   a transformer such that the same instance is not shared between different
>   transformers.

Reviewer: ```suggestion
* Fixes a bug in :class:`TableVectorizer` with `remainder`: it is now cloned if it's
  a transformer so that the same instance is not shared between different
  transformers.
```
